src/modules/Auth/components/ButtonMenu.tsx

```diff
@@ -0,0 +1,95 @@
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import Logout from '@mui/icons-material/Logout';
+import Settings from '@mui/icons-material/Settings';
```

Comment:
Add suffix Icon in the end for all icons
```suggestion
import SettingsIcon from '@mui/icons-material/Settings';
```

Reply:
eslint prefer this order :D